appveyor.yml

```diff
@@ -15,6 +16,11 @@ skip_tags: true
 
 clone_folder: c:\projects\habitat
 
+environment:
+  matrix:
+    - hab_build_action: "test;build"
```

Comment:
Any reason why not to break out the matrix like:

```
- hab_build_action: "build"
- hab_components: "hab"

- hab_build_action: "test"
- hab_components: "hab"

- hab_build_action: "test"
- hab_components: "http-client"

etc, etc
```

Something like this would allow the appveyor.ps1 to be much simpler (no looping or string parsing) and the test actions to be more clear.